data/projects/p/paraswap.yaml

```diff
@@ -7,6 +7,9 @@ blockchain:
  - address: "0x59C7C832e96D2568bea6db468C1aAdcbbDa08A52"
```

Comment:
Sorry @mounibec - I missed this, but your need to indent the address. YAML is very fussy about indentations 😓